### PR TITLE
Add FLUSH DB commands to swss service missed during merge

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**

### CODE REVIEW

**Comment by oleksandrivantsiv** [comment]
swss.service won't start after device boot due to issue https://github.com/Azure/SONiC/issues/50
To start swss service the following command should be executed:
`systemctl restart swss syncd`

**Comment by lguohan** [comment]
also discussed this internally, feel we should have swss.service that control both swss docker and syncd docker now and remove syncd.service for now. 

once the swss hitless restart is implemented then we can make them a separate service.

**Comment by lguohan** [comment]
@qiluo-msft, please also comment

File: files/build_templates/swss.service.j2 [hunk 1]
```diff
@@ -1,10 +1,13 @@
 [Unit]
-Description=orchagent container
+Description=switch state service container
 Requires=database.service
 After=database.service
 
 [Service]
 User={{ sonicadmin_user }}
+/usr/bin/docker exec database redis-cli -n 0 FLUSHDB
```
Comment (lguohan) [inline files/build_templates/swss.service.j2]: ExecStartPre=

Comment (oleksandrivantsiv) [inline files/build_templates/swss.service.j2]: Don't know how I missed it :)

**Comment by qiluo-msft** [comment]
@oleksandrivantsiv We found a bug in sonic-mgmt repo that the FLUSHDB timing is actually incorrect. Please wait for https://github.com/Azure/sonic-mgmt/pull/112/files and follow it if approved.

**Pull request merged**